link-user-to-episys-name-record/REPWRITERSPECS/BANNO.DATABASE.CHECK.V1.POW

```diff
@@ -969,7 +1134,15 @@ PROCEDURE POLLTRACKING
   END
  TRACKINGLISTCOUNT=0
 
+[* USERCHAR12 is the preference Locator and connects a specific tracking 8 to
+** the member logged in. USERCHAR12 is only filled in for BANNO use.
+*]
+
+ BNODEBUGMSG="ACCOUNT: "+ACCOUNT:NUMBER
+ CALL BNODEBUGMSGADD
+
  FOR EACH TRACKING WITH (TRACKING:TYPE=8 AND
+                         VALUE(TRACKING:USERCHAR12)>0 AND
```

Comment:
Hi Tom,
 I think adding the check for VALUE(TRACKING:USERCHAR12)>0 before loading the tracking in the tracking array to be evaluated will prevent NetTeller Tracking records from getting updated with banno log in information. I remember the Idea was to recycle NetTeller tracking 8 records. NetTeller stores that Value for Preference Locator in 
USERNUMBER1

Reply:
Looking at it more deeply, it appears the old code was looking for a matching TRACKING:USERCHAR12, and if not found it just created a tracking 8. So it looks like we have been making Banno tracking 8, regardless of  if they have a NetTeller tracking 8. Is that better in some way?